docusaurus/src/theme/MDXComponents/index.js

```diff
@@ -21,4 +21,5 @@ export default {
   SpecSchema,
   PyAirbyteExample,
   ProductInformation,
+  Details,
```

Comment:
I'm surprised that this is all that is needed to replace the default `details` component!

Is that something specific to docusaurus, such that you just need to have a component named the same (but with a capitalized first letter)?

Reply:
Me too! In fact, I created the component and use it on the .md files with <Details> instead of <details>, but I realized that other elements in the page were also changing without changing the component used.

I thought this will be handy so people don't have to remember using the JSX component instead of the "plain" html one. But if you feel it's confusing, I can rename the component so we still have both!

Reply:
I think is actually a Markdown thing. It seems that both `<Details>` and `<details>` will render become `<details>`.

Eg: <Details> will be rendered automatically as <details></details> </Details>


This was easy to spot on docusaurus by logging `MDXComponents from "@theme-original/MDXComponents"`;  and our CustomComponents export.
I'm actually overwriting the definition of `<Details>`  by using the same name, which it turns out it overwrites both.

I think this is useful in this case, because this is the default behaviour that we want for this component, but let me know if you would prefer to have a separate component!

https://www.loom.com/share/f1842ecd722a43bdb3e878ac0888949b?sid=a1ed795d-70b5-499c-ad6c-ea12dc7fbb27

Reply:
No this makes sense, I am pleasantly surprised that the overriding is this easy to configure 👍